understanding/22/focus-appearance-minimum.html

```diff
@@ -197,15 +198,15 @@ <h3>Adjacent contrast</h3>
         </section>
         <section id="unobscured">
             <h3>Not fully obscured</h3>
-            <p>Where other content can overlap with a focused item, the focused element should not be hidden. Typical content which can overlap focused items are sticky footers, sticky headers, or non-modal dialogues. As a user tabs through the page, these layers of content can obscure the focused item, including the focus indicator.</p>
-            <p class="note">The criterion specifies the "focused item" in the 'Not fully obscured' bullet rather than the "focus indicator". The requirements for the indicator are set in the previous bullets, the last bullet applies to the item as a whole.</p>
+            <p>Where other content can overlap with a focused item, the focused element should not be hidden. Typical types of content that can overlap focused items are sticky footers, sticky headers, or non-modal dialogues. As a user tabs through the page, these layers of content can obscure the focused item, including the focus indicator.</p>
+            <p class="note">The criterion specifies the "focused item" in the 'Not fully obscured' bullet rather than the "focus indicator". The requirements for the indicator are set in the previous bullets; the last bullet applies to the item as a whole.</p>
 
         </section>
         <section id="relation-non-text-color-contrast">
-            <h3>Relation to non-text color contrast</h3>
-            <p><a href="non-text-contrasast" class="sc">Success Criterion 1.4.11 Non-text contrast</a> requires that a UI component maintains contrast across it’s states, including when focused. For example, a checkbox must contrast with its adjacent colors, and the check within the checkbox must contrast with its adjacent colors. </p>
+            <h3>Relationship to Non-text Contrast</h3>
+            <p><a href="non-text-contrast" class="sc">Success Criterion 1.4.11 Non-text Contrast</a> requires that a UI component maintains contrast against the adjacent <em>background</em> for each of its states. When it has does not have focus, it must be visible against the background. When it receives focus, it still must be visible. However, Non-text Contrast does require contrast <em>between</em> the UI component's focused and non-focused states. Merely that in each state it contrasts against the adjacent background. </p>
```

Comment:
Missing a "not" from "However, Non-text Contrast does NOT require"

Reply:
added in a new commit.